docs/06-objects-of-interest/06.01-tiffs.md

```diff
@@ -0,0 +1,18 @@
+# TIFF files
+
+Tiff files have special handling for importing.
+
+Like all files, we upload the original, but create an optimised and thumbnail
+png version.
+
+For TIFF files, these pngs derive from the first image file found inside the
+container format, which, in a file named `xxx.tif` will explode out as
+`xxx.png` or `xxx.jpg`.
```

Comment:
What determines what type it is? Presumably there are many TIFFs that are baseline tiffs and do not contain a png or jpeg?

Reply:
Looks like TIFF is an absolute minefield having skimmed part of https://en.wikipedia.org/wiki/TIFF

TIL that TIFF was given a mockronym of Thousands of Incompatible File Formats.

Reply:
Wait until you see [PSD](https://github.com/gco/xee/blob/master/XeePhotoshopLoader.m#L108). For a strategy of what to do (and not), see this [tropical jungle](https://github.com/guardian/grid/issues/2395#issuecomment-503501457). Because _forest_ would be an understatement.

But, coming back to this comment in code, it isn’t true. TIFF is not a container format (forgetting for now that it can store pixel data with a JPEG compression which is irrelevant here). We ask ImageMagick to convert a TIFF to a PNG explicitly. And if it has layers, ImageMagick returns multiple files for each.

Reply:
Right! That is absolutely not clear from the code or these docs.

Reply:
That's not a new change.  That's the change we (I) made in April that actually works.

Reply:
This documentation is new though and it is misleading. I inferred what the documentation says from the comments on the function too - so those comments are also misleading.

Reply:
Ah, I see.  Today-I-Learn...